db/migrate/20200122143238_add_status_to_review_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToReviewRequest < ActiveRecord::Migration[6.0]
+  def change
+    add_column :review_requests, :status, :string
```

Comment:
shouldn't we use an `enum` for statuses?

Reply:
Done!